draft-bbs-signatures.md

```diff
@@ -366,21 +378,25 @@ Procedure:
 
 1. (W, H0, H) = octets_to_point(PK)
 
-2. h = XOF(SK  || msg[i] || ... || msg[L])
+2. sig_dst = OS2IP(HASH(PK || L || H_s || H_d || H_1 || ... || H_L || CipherInfo)) mod q
```

Comment:
Can we also add in the ciphersuite ID here too? as the first argument? e.g the unique string that identifies the cipher suite

Reply:
Done! Just a note, Ciphersuite_ID is not currently defined in ciphersuites. Will make an additional PR to add the definition.